code/controllers/subsystem/SSticker.dm

```diff
@@ -419,12 +419,10 @@ SUBSYSTEM_DEF(ticker)
 					flick("station_explode_fade_red", cinematic)
 					SEND_SOUND(world, sound('sound/effects/explosion_distant.ogg'))
 					cinematic.icon_state = "summary_selfdes"
-			stop_delta_alarm()
 
 	//If its actually the end of the round, wait for it to end.
 	//Otherwise if its a verb it will continue on afterwards.
 	spawn(300)
-		stop_delta_alarm() // If we've not stopped this alarm yet, do so now.
```

Comment:
Just verify this is still needed to be removed with the delta alarm changes please